Gems/AWSGameLift/Code/AWSGameLiftClient/Source/AWSGameLiftClientLocalTicketTracker.h

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/std/parallel/mutex.h>
+#include <AzCore/std/parallel/thread.h>
+
+#include <Request/IAWSGameLiftMatchmakingInternalRequests.h>
+
+#include <aws/gamelift/model/MatchmakingTicket.h>
+
+namespace AWSGameLift
+{
+    enum TicketTrackerStatus
+    {
+        Idle,
+        Running
+    };
+
+    //! AWSGameLiftClientLocalTicketTracker
+    //! GameLift client ticket tracker to describe submitted matchmaking ticket periodically,
+    //! and join player to the match once matchmaking ticket is complete.
+    //! All games in public release should use notifications regardless of volume.
```

Comment:
IMHO this is still not clear "For use in production, please see GameLifts guidance about matchmaking at volume. The continuous polling. approach here is only suitable for low volume matchmaking and is meant to aid with development only"

Reply:
Will update